code/graph.q

```diff
@@ -94,6 +94,7 @@ graph:.ml.connectEdge[graph;`featureDescription ;`symEncode      ;`preprocParams
 graph:.ml.connectEdge[graph;`featureCreation    ;`creationTime   ;`preprocParams;`creationTime]
 graph:.ml.connectEdge[graph;`featureSignificance;`sigFeats       ;`preprocParams;`sigFeats]
 graph:.ml.connectEdge[graph;`labelEncode        ;`symMap         ;`preprocParams;`symMap]
+graph:.ml.connectEdge[graph;`runModels          ;`metaData       ;`preprocParams;`metaData]
```

Comment:
It probably makes more sense to add this to the predictParams node rather than the preprocessing? Looking at the graph and workflow we should try to get nodes to complete as early as they can such that if something goes wrong they'll have been completed and we can retrieve relevant information as necessary. This will necessitate a change to the graph draw.io and png as well as updating inputs for a couple of the other nodes